easybuild/easyconfigs/a/ASAP3/ASAP3-3.13.7-foss-2023a-ASE-3.24.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "PythonPackage"
+
+name = 'ASAP3'
+version = '3.13.7'
+_aseversion = '3.24.0'
+versionsuffix = '-ASE-' + _aseversion
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': False}
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5501132872dc79ea5d4a4d06f928a73ca20059ec7e4f37e9b7cd111f974fd57c']
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('ASE', _aseversion),
+    ('kim-api', '2.3.0'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
```

Comment:
```suggestion

```

Remove to pass the CI test (these are now set in the easyblock by default)

Reply:
Done